app/controllers/Subscriptions.scala

```diff
@@ -63,11 +64,14 @@ class Subscriptions(
 
   def digital(countryCode: String): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: Settings = settingsProvider.settings()
-    val title = "Support the Guardian | Digital Subscription"
+    val title = "Support the Guardian | Digital Pack Subscription"
     val id = "digital-subscription-landing-page-" + countryCode
     val js = "digitalSubscriptionLandingPage.js"
     val css = "digitalSubscriptionLandingPageStyles.css"
-    Ok(views.html.main(title, id, js, css)).withSettingsSurrogateKey
+    val description = Some(stringsConfig.digitalPackLandingDescription)
```

Comment:
Description here wants to be wrapped in `Option` rather than `Some`. This is because when we call `stringsConfig.digitalPackLandingDescription` it calls  `config.getString("digitalPackLanding.description")` which can return the String config item we want or if it is not present will throw an exception. See: https://lightbend.github.io/config/latest/api/com/typesafe/config/Config.html#getString-java.lang.String-.

Ideally, this behaviour would be modelled at the type level in `StringConfig` the object interacting with the TypeLevel config object but out of scope for this PR.